app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/ViewerFragment.java

```diff
@@ -72,6 +73,7 @@
 
     private VectorLayer mSentLocationsLayer;
     private VectorLayer mUsersLocationsLayer;
+    private VectorLayer mReceivedLocationsLayer;
```

Comment:
We  should move all the layers to be part of the View-model/model and "plug" them to the map on map initialization. 
If we'll do it, we can update the map by manipulating the entities at the view-model/model layers without messing around with with the fragments/activities.
I'm not asking you to change it all now, just for your thoughts. 
Lets talk about it tomorrow and decide on a future refactoring.